database/migrations/2024_08_30_142048_add_article_type_column_to_digital_publication_articles_table.php

```diff
@@ -10,12 +10,12 @@
 {
     public function up(): void
     {
-        $types = array_map(fn($enum) => $enum->value, DigitalPublicationArticleType::cases());
-        Schema::table('digital_publication_articles', function (Blueprint $table) use ($types) {
-            $table->enum('article_type', $types)->default('text');
+        $categories = array_map(fn ($category) => $category->value, DigitalPublicationArticleCategory::cases());
+        Schema::table('digital_publication_articles', function (Blueprint $table) use ($categories) {
+            $table->enum('category', $categories)->nullable(false)->default('text');
         });
         foreach (DigitalPublicationArticle::withTrashed()->get() as $article) {
-            $article->article_type = $article->type;
+            $article->category = $article->type;
```

Comment:
All the `category`s after this migration is run are set to the default value of `text`. This migration is trying to copy values over from a column called `->type` which doesn't exist. Change references of `type` to `article_type` in this migration.

Reply:
I've since created this [subsequent migration](database/migrations/2024_09_05_105828_alter_category_column_on_digital_publication_articles_table.php) that changes `article_type` -> `category`, since the test DB already has run the first migration with `type` -> `article_type`.